.github/workflows/generate_sample.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate the sample project
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  test:
+    name: Generate the sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
```

Comment:
The latest Flutter stable version is `3.10.1` according to the [release doc](https://docs.flutter.dev/release/archive?tab=macos). Should we update this field?

Reply:
@nmint8m It does not belong to this PR scope. We might consider updating in the following https://github.com/nimblehq/flutter-templates/pull/201 🙏